cli/src/lib/generateManifests.js

```diff
@@ -93,11 +115,20 @@ module.exports = (paths, config, publicUrl) => {
             },
         ],
         start_url: '.',
-        display: 'browser',
+        display: 'standalone',
```

Comment:
What's the impact of this change?  Seems a bit unrelated?

Reply:
It is unrelated, but easy to fix - see https://dhis2.atlassian.net/browse/LIBS-355